### DISCUSSION TITLE
Settings: "Controller, Action & Template" is not used on change

### DISCUSSION THREAD

**Discussion by akno77** [pimcore/pimcore#7994] [Q&A / Support]
Hi,

I have the following problem with Pimcore v6.8.7:

1. I create a new blank page
2. In the page settings I set a new Controller & Action and click "save"
3. The page keeps showing the blank page content and I am not able to edit the content of the new Action Controller Template (Twig)
4. If I Publish the page I can see the new content and I am able to edit it.
5. If the new assigned Controller & Action Template has some required editable fields I am not able to Publish the page: "Prevented publishing document - missing values for required editables".
6. So if I wanna change the template for a page with required editable fields I am not able to get it published or editable!

Is this Pimcore "normal" behavior or do I miss sth.?

**Comment by NiklasBr** [comment]
Yes, this is unfortunately the current behaviour, you also need to press the *reload* button in the UI after saving to see the changes.
